cmd/vicadmin/html

```diff
@@ -0,0 +1 @@
+../../isos/vicadmin
```

Comment:
HTML template files go in this directory and are uploaded in the /home/vicadmin directory on the ISO.